view/templates/event_stream_item.tpl

```diff
@@ -22,7 +22,7 @@
 	<div class="event-location">
 		<span class="event-label">{{$location_label}}</span>&nbsp;
 		{{if $location.name}}
-		<span class="location">{{$location.name}}</span>
+		<span class="location">{{$location.name nofilter}}</span>
 		{{/if}}
 		{{if $location.map}}{{$location.map}}{{/if}}
```

Comment:
The map needs to be unescaped, it contains some HTML to show a map.

Reply:
I see a description, should it be unescaped as well?

Reply:
Best is to have a look at ```src/Model/Event.php``` for this template. I just had a random look at this PR by now. I guess I will need some hours to really make a better check.